scripts/quests/windurst/Let_Sleeping_Dogs_Lie.lua

```diff
@@ -0,0 +1,257 @@
+                    if quest:getVar(player, 'Prog') == 4 then
+                        return quest:event(506)
+                    else
+                        return quest:event(488)
+                    end
+                end,
+            },
+
+            ['Chomoro-Kyotoro'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 4 then
+                        return quest:event(501)
+                    else
+                        return quest:event(489)
+                    end
+                end,
+            },
+
+            ['Paku-Nakku'] =
+            {
+                onTrade = function(player, npc, trade)
+                    if
+                        npcUtil.tradeHasExactly(trade, xi.item.BUNCH_OF_BLAZING_PEPPERS) and
+                        quest:getVar(player, 'Prog') == 1
+                    then
+                        return quest:progressEvent(494)
+                    end
+                end,
+
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 3 then
+                        return quest:progressEvent(499)
+                    elseif quest:getVar(player, 'Prog') == 4 then
+                        return quest:event(500)
+                    else
+                        return quest:event(482)
+                    end
+                end,
+            },
+
+            ['Pechiru-Mashiru'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 2 then
+                        return quest:progressEvent(495, 0, 4155, 1102, 1)
+                    elseif quest:getVar(player, 'Prog') == 3 then
+                        return quest:event(496)
+                    elseif quest:getVar(player, 'Prog') == 4 then
+                        return quest:progressEvent(497)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [494] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 2)
+                end,
+
+                [495] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 3)
+                end,
+
+                [497] = function(player, csid, option, npc)
+                    quest:complete(player)
+                end,
+
+                [499] = function(player, csid, option, npc)
+                    if option == 2 then
+                        quest:setVar(player, 'Prog', 4)
+                    end
+                end,
+            },
+        },
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            ['Maabu-Sonbu'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 0 then
+                        return quest:progressEvent(319, 653392653, 4155, 1102, 1020) -- 0, 0, 0, 0)
```

Comment:
Can you please remove these commented values? Known that default is 0 for all params not specified.  Also, can you take a look into 4155, 1102, and 1020, as they smell like either keyitems or items.

This applies to all of the event parameters below as well.

Reply:
Swapped in item vars! Thanks!